src/css/index.scss

```diff
@@ -0,0 +1,105 @@
+section.contact {
+  padding-bottom: 100px;
+  padding-top: 10px;
+}
+section.contact h2 {
+  font-size: 38px;
+  font-size-adjust: inherit;
```

Comment:
`font-size-adjust: inherit;` take no effect and it is not a valid css property.
Where did you get this from?